Mathlib/Probability/Distributions/Uniform.lean

```diff
@@ -172,9 +185,32 @@ theorem integral_eq (huX : IsUniform X s ℙ) :
 
 end IsUniform
 
-end pdf
+variable {X : Ω → E}
+
+lemma IsUniform.uniformMeasure {s : Set E} :
+    IsUniform (id : E → E) s (ProbabilityTheory.cond μ s) μ := by
```

Comment:
The name does not seem to describe the result?

Reply:
I'll make the necessary changes, do you want to take it off the queue?

Reply:
"maintainer merge" asks for a final validation from a maintainer, it does not send the PR to bors right away, so it's not on any queue right now: you can make the changes in this PR, and I'll merge it afterwards.

Reply:
Great, I've made all the necessary changes I think!